rulebook.tex

```diff
@@ -250,83 +278,103 @@ \section{Introduction}
  \mbox{\url{https://github.com/robocup-logistics/rcll-rulebook}}}.
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \subsection{The Task}
 \label{sec:task}
-
-Our aim is to provide a simplified Smart Factory environment. The
-teams must complete the following task without human interference,
-competing with a second team against the clock. In the RCLL,
-autonomous robot agents have to handle the logistics of materials
-through several (dynamic) stages to produce final goods to fulfill
-orders. The machines are specific MPS stations.
-
-The Logistics League's main challenge is a multistage production cycle
-of different product variants with self-crafted intermediate products
-and delivery of final products. This genuine goal will be rewarded
-considerably higher than partial fulfillment of the task. Autonomous
-robots transport small products between the processing
-machines. Machines are MPS stations which complete a particular
-refinement step like mounting a colored ring or top-most cap. If
-procurable this will lead to a new sub-assembly or final product
-(cap). Complete work orders require all related sub-assemblies of
-product variants (cf. \refsec{sec:production-complexities}).
-
-This work flow is controlled by a referee box broadcasting information
-via wifi (see \refsec{sec:referee-box}). The work flow itself is
-divided into three different phases: a setup
-phase (see \refsec{sec:setup-phase}), an exploration phase (see
-\refsec{sec:exploration-phase}) during which robots receive scores for
-correctly discovering and publishing the positions of the yet
-unknown different machines on the field. After this phase the referee
-box will announce all machine types and designations. In the following
-production phase (see~\refsec{sec:production-phase}) orders are
-announced by the referee box, which the robots must fulfill
+ Our aim is to provide a simplified Smart Factory environment, where
+ teams of autonomous robots must operate a flexible production plant without
+ human interference and against the clock.
+ The Logistics League's core task is to fulfill placed product orders by
+ executing multi-stage production cycles, before delivering the finished
+ products within the requested time frame.
+ This is achieved by transporting small pieces of material between assembly
+ machines (\ac{MPS} stations, see \refsec{sec:machines}),
+ which can be instructed to perform different refinement steps,
+ such as mounting a colored ring on a cylindrical base product.
+ Efficient handling of the material logistics and utilization of the available
+ machines are core requirements to succeed in the \ac{RCLL}.
+
+ This work flow is controlled by a \acf{refbox}~\cite{RCI-RefBox}%
+\footnote{Code and documentation available at
+\url{https://github.com/robocup-logistics/rcll-refbox}}
+broadcasting information via wifi (see \refsec{sec:referee-box}).
+It is a software system that runs on a
```

Comment:
What system is provided by the OC that the refbox runs on? I am rather familiar with the league, but have no idea what is meant here.

Reply:
I thought this means that the OC is responsible to provide the laptop where the refbox runs on? Is that how it is supposed to be?